Chinese/火星情报局/汪涵开始主持.md

```diff
@@ -0,0 +1 @@
+汪涵说：”欢迎大家收看火星情报局“
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.